core/new-gui/src/app/dashboard/type/workflow-version-entry.ts

```diff
@@ -0,0 +1,5 @@
+export interface WorkflowVersionEntry extends Readonly<{
+  vId: number|undefined;
+  creationTime: number|undefined;
+  content: string | undefined;
```

Comment:
the format looks not good (spaces).

also, maybe use `vId?: number`? (so as other optional entries). 

meanwhile, are all of them really "optional"? I thought each version "should" have a vId, a creationTime, and a content. Maybe all three attributes should all be not optional? just a thought.